layouts/system76/bonw14/layout.csv

```diff
@@ -1,102 +1,102 @@
-K00,6,0
```

Comment:
Was this intentional? The bonobo has a different mapping

Reply:
Ah. I'll have to see why it's not generating the same thing as `layouts.sh` here.

Reply:
Apparently the `/* */` comments were being handled incorrectly. Added a regex replacement to remove that, and an assertion.